virtual-machine/encrypt-vm/encrypt-windows-vm.ps1

```diff
@@ -6,7 +6,7 @@ $rgName = "myResourceGroup"
 #Region
 $location = "East US"
 #Password to place w/in the KeyVault
-$securePassword = ConvertTo-SecureString -String "P@ssword!" -AsPlainText -Force
+$securePassword = ConvertTo-SecureString -String "Password@16digits" -AsPlainText -Force
```

Comment:
Only use user-supplied or securely randomly generated passwords in example scripts.

Reply:
@Daya-patil 

| Requirement | Issue | Fix |
|---------------|-------|----|
| New passwords are randomly generated | A static password is in the script | Instead of providing a password as a string, please use `(New-Guid).Guid` to generate a password for the user. |